server/tm_cmds.go

```diff
@@ -10,8 +12,25 @@ import (
 	"sigs.k8s.io/yaml"
 
 	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
 	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	"github.com/cosmos/cosmos-sdk/version"
+	authblock "github.com/cosmos/cosmos-sdk/x/auth/block"
```

Comment:
any reason these cant be in server as well?

Reply:
I can move this to server - `"github.com/cosmos/cosmos-sdk/x/auth/block"` but the ``` "github.com/cosmos/cosmos-sdk/types/query"
	"github.com/cosmos/cosmos-sdk/version" ``` are generated by `proto` - not sure how to get them to be generated elsewhere...?